partner_contact_role/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_res_partner_role_public,res_partner_role group_public,model_res_partner_role,base.group_public,1,0,0,0
+access_res_partner_role_portal,res_partner_role group_portal,model_res_partner_role,base.group_portal,1,0,0,0
```

Comment:
I think you can just omit the group if your plan is giving everybody read access. That way you can remove these 2 lines...